composer.json

```diff
@@ -6,27 +6,27 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.2",
-        "drupal/core": "^8.7",
-        "drupal/language_selection_page": "2.3",
+        "php": ">=7.3",
+        "drupal/core": "^8.9 || ^9",
+        "drupal/language_selection_page": "^2.6",
         "drupal/pathauto": "^1.6"
     },
     "require-dev": {
         "composer/installers": "~1.5",
         "consolidation/annotated-command": "^2.8.2",
         "consolidation/robo": "~1.4",
-        "drupal/core-composer-scaffold": "^8.8",
+        "drupal/core-composer-scaffold": "^8.9 || ^9",
         "drupal/config_devel": "~1.2",
         "drupal/devel": "~1.2",
         "drupal/drupal-extension": "~4.0",
         "drupal/redirect": "^1.4",
-        "drush/drush": "~9.0@stable",
+        "drush/drush": "~10.0",
         "guzzlehttp/guzzle": "~6.3",
         "openeuropa/behat-transformation-context": "~0.1",
-        "openeuropa/code-review": "^1.3.0",
-        "openeuropa/drupal-core-require-dev": "^8.7",
-        "openeuropa/task-runner": "~1.0.0-beta6",
-        "phpunit/phpunit": "~6.0",
+        "openeuropa/code-review": "~1.6",
+        "openeuropa/drupal-core-require-dev": "^8.9 || ^9",
+        "openeuropa/task-runner-drupal-project-symlink": "^1.0",
+        "phpunit/phpunit": "^7 || ^8",
         "symfony/browser-kit": "~3.0 || ~4.0"
```

Comment:
Do we need `"symfony/browser-kit": "~3.0 || ~4.0"`?

Reply:
The same as above.

Reply:
"symfony/browser-kit" is included in the package's dependencies. Why do we need to point it here?

Reply:
Yes, as long as tests pass we can remove dependencies.